docs/source/backends/create_your_backend.rst

```diff
@@ -93,3 +93,41 @@ The previous example can be used with the following class definition
             converter = []
             parser = BaseParser
             renderer = BaseRenderer
+
+Once you have your python package configured with the correct entry points
+you should have a directory tree that looks like this.
+
+.. code-block:: bash
+
+    $ tree example_backend
+    .
+    ├── example_backend
+    │   └── __init__.py
+    └── setup.py
+
+And now you can install your package using `pip install example_backend`.
```

Comment:
This assumes the package is published on pypi, which is something most users won't do: a very small amount is able or willing to do that; the users who haven't clear how to publish packages on pypi won't understand this passage.

I'd write something like:

```
Now you can install your package using ``pip install -e ./example_backend``.
```

Reply:
I always showed examples with the following directory

```
example_backend
  - example_backend
    - __init__.py
  - setup.py
```

as if the user is in the parent directory so `pip install example_backend` make sense as pip picks up the rigth folder but `pip install -e ./example_backend` should be fine